FGO-py/fgoDetect.py

```diff
@@ -109,7 +110,7 @@ def isCardSealed(self):return[any(self._compare(j,(28+257*i,444,234+257*i,564),.
     def isFpContinue(self):return self._compare(self.tmpl.FPCONTINUE,(646,639,883,707))
     def isFpSummon(self):return self._compare(self.tmpl.FPSUMMON,(643,20,812,67))
     def isFriendListEnd(self):return self._isListEnd((1255,709))
-    def isHouguReady(self,that=None):return(lambda that:[not any(that._compare(j,(313+231*i,172,515+231*i,258),.52)for j in(self.tmpl.HOUGUSEALED,self.tmpl.CHARASEALED,self.tmpl.CARDSEALED))and(numpy.mean(self._crop((144+319*i,679,156+319*i,684)))>55 or numpy.mean(that._crop((144+319*i,679,156+319*i,684)))>55)for i in range(3)])((time.sleep(.15),type(self)())[1]if that is None else that)
+    def isHouguReady(self,that=None):return(lambda that:[not any(that._compare(j,(313+231*i,172 if XDetect.region != 'NA' else 194,515+231*i,258),.52)for j in(self.tmpl.HOUGUSEALED,self.tmpl.CHARASEALED,self.tmpl.CARDSEALED))and(numpy.mean(self._crop((144+319*i,679,156+319*i,684)))>55 or numpy.mean(that._crop((144+319*i,679,156+319*i,684)))>55)for i in range(3)])((time.sleep(.15),type(self)())[1]if that is None else that)
```

Comment:
NO `if XDetect.region != 'NA'` here, override it in XDetectNA

Reply:
Considering that the command card moves up and down periodically when it is displayed, if you think there is a real need to change 172 to 194, should you also adjust 258?

Reply:
Done and adjusted. When I tested even with the card movement at it's highest position this was the position of the "NP Sealed"

![max-hougusealed](https://github.com/hgjazhgj/FGO-py/assets/17742637/1a240a26-89c1-4c42-8174-3d506d7868be)

I think it's better to set it to 194.